src/enkf/gridio_fv3reg.f90

```diff
@@ -680,6 +682,58 @@ subroutine writegriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,vargrid
     !======================================================================
 
   end subroutine writegriddata
+  subroutine writeincrement(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,grdin,no_inflate_flag)
```

Comment:
Please add brief description of this subroutine

Reply:
Done!